packages/modules/itwin-error-handling-react/package.json

```diff
@@ -76,7 +76,7 @@
     "typescript": "^3.7.2"
   },
   "peerDependencies": {
-    "@bentley/ui-core": "^2.2.1",
+    "@bentley/ui-core": "^2.3.3",
```

Comment:
shouldn't this also be a dev dep too?

Reply:
> shouldn't this also be a dev dep too?

updated for consistency, though it shouldn't really be necessary since we are always developing in the context of an app